doc/md_src_files/055_decision-points_2.md

```diff
@@ -205,13 +205,13 @@ A **mission essential function (MEF)** is a function “directly related to acco
 | MEF Failure            | Any one mission essential function fails for period of time longer than acceptable; overall mission of the organization degraded but can still be accomplished for a time |
 | Mission Failure        | Multiple or all mission essential functions fail; ability to recover those functions degraded; organization’s ability to deliver its overall mission fails                |
 
-#### Advice for Gathering Information to Answer the Mission Impact Question
+### Advice for Gathering Information to Answer the Mission Impact Question
 
```

Comment:
How about:
Mission Impact Risk Factors

Reply:
I'm not sure that we are enumerating risk factors in this section.

Reply:
My thinking is: The information that is being gathered would include indicators of risk. 
If you disagree, consider a shorter title than "Advice for Gathering Information to Answer the Mission Impact Question".  what about, 'Information for Mission Impact'?